__tests__/api.test.js

```diff
@@ -114,4 +114,49 @@ describe('/get/reviews/:review_id/comments', ()=>{
             expect(err.body.msg).toBe('Bad Request!')
         })
     })
-})
\ No newline at end of file
+})
+
+describe('POST api/reviews/:review_id/comments', ()=>{
+    test('returns 201', ()=>{
+        const commentToPost = {
+            body: 'test',
+            username: 'philippaclaire9',
+        }
+        return request(app).post('/api/reviews/2/comments').send(commentToPost).expect(201)
+        })
+    test('returns the new comment data', ()=>{
+            const commentToPost = {
+                body: 'test',
+                username: 'philippaclaire9',
+            }
+            return request(app).post('/api/reviews/2/comments').send(commentToPost).expect(201).then((response)=>{
+                const output = response.body.comment[0]
+                expect(output).toHaveProperty('comment_id', expect.any(Number))
+                expect(output).toHaveProperty('body', expect.any(String))
+                expect(output).toHaveProperty('review_id', expect.any(Number))
+                expect(output).toHaveProperty('author', expect.any(String))
+                expect(output).toHaveProperty('votes', expect.any(Number))
+                expect(output).toHaveProperty('created_at', expect.any(String))
+            })
+        })
+    test('responds with 404 not found when no reviews are found', ()=>{
+        const commentToPost = {
+            body: 'test',
+            username: 'philippaclaire9',
+        }
+        return request(app).post('/api/reviews/100/comments').send(commentToPost).expect(404)
+        .then((err)=>{
+            expect(err.body.msg).toBe("You can't comment on a review that doesn't exist!")
+        })
+    })
+    test('responds with 400 bad request when passed an invalid id', ()=>{
+        const commentToPost = {
+            body: 'test',
+            username: 'philippaclaire9',
+        }
+        return request(app).post('/api/reviews/notAReview/comments').send(commentToPost).expect(400)
+        .then((result)=>{
+            expect(result.body.msg).toBe("Invalid Id, please enter a number.")
+        })
```

Comment:
nice work so far but could you add a couple of tests for if the comment to post has an invalid key or is an empty object etc 
`{notBody: "test"
   username: "whatever"
   }`